assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/bamtofastq" target="_blank">nf-core/bamtofastq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/bamtofastq/tree/dev" target="_blank">nf-core/bamtofastq</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/bamtofastq/2.0.0/docs/output" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/bamtofastq/dev/docs/output" target="_blank">documentation</a>.
```

Comment:
Yes, this will be passing the linting from the tools version 2.10 onwards :) The old links are not working and are already fixed in the dev version :)